common/src/main/java/com/wynntils/sockets/model/HadesModel.java

```diff
@@ -127,6 +135,22 @@ public void onPartyListUpdate(RelationsUpdateEvent.PartyList event) {
 
     @SubscribeEvent
     public void onWorldStateChange(WorldStateEvent event) {
+        if (event.getNewState() == WorldStateManager.State.WORLD && isSocketOpen() && firstWorldJoin) {
+            firstWorldJoin = false;
+
+            // FIXME: Use the proper reload command here, once they are reworked
+            MutableComponent failed = Component.literal("Welps! Trying to connect to Hades failed.")
+                    .withStyle(ChatFormatting.GREEN);
```

Comment:
These literals should really be translatable components. Also "Welps" is a) informal and b) probably should be welp, don't know what I was doing when I wrote that.

Reply:
We can fix that now or later. I think we need to do a full sweep of the entire Artemis code base and fix all non-i18n strings. (The one exception is in log messages; they don't need to be i18n)

Reply:
I opened https://github.com/Wynntils/Artemis/issues/906 for this.